test/e2e/webdriver/index.js

```diff
@@ -14,6 +14,8 @@ async function buildWebDriver ({ responsive, port } = {}) {
 
   const driver = new Driver(seleniumDriver, browser)
 
+  await driver.delay(600)
```

Comment:
```suggestion
  await driver.delay(1000)
```
Maybe it just needs more time? :man_shrugging: